incubator/istio/Chart.yaml

```diff
@@ -3,8 +3,8 @@ description: Istio Helm chart for Kubernetes
 name: istio
 # To avoid confusion the helm chart version stays in sync with the istio version - DO NOT UNSYNC -- ldemailly@google.com
 # https://github.com/kubernetes/charts/issues/1501
-version: 0.1.6-chart6
-appVersion: 0.1.6
+version: 0.2.6-chart1
```

Comment:
Istio users were getting confused when trying to install because helm shows the chart version not the appVersion. So while they wanted to install 0.1.6 of Istio they were searching and finding 0.3.0 of the Helm chart. There is no rule against versioning this way if its what the maintainers of the chart want.